src/analyzer/analyzergain.h

```diff
@@ -15,18 +15,18 @@ class ReplayGain;
 
 class AnalyzerGain : public Analyzer {
```

Comment:
Ok -- in my VAMP removal branch I introduce a concept of plugins for each individual algorithm which the analyzers then use so I'll do this cleanup in my branch.